packages/eslint-config/react.js

```diff
@@ -34,6 +34,11 @@ module.exports = {
                         importNames: ["styled"],
                         message: "Please use styled from @mui/material/styles instead.",
                     },
+                    {
+                        name: "@mui/material",
+                        importNames: ["Badge"],
+                        message: "Please use Badge from @comet/admin instead",
+                    },
```

Comment:
@jomunker @johnnyomair I don't think we can just replace MUIs Badge here with our one, as they have different use cases. 
MUIs Badge is used as a wrapper around something to show the Badge to the top right of that item. 
Our Badge seems to be for standalone use only. 

Should we maybe add an additional variant to the existing MUI Badge instead of creating our own?
Should we use our own and use this "standalone" behavior only when there are no children?

Do we actually need this component anywhere other than the menu?
Maybe it makes more sense to just add a `notificationCount: number` prop to `MenuItem`, which does the rest internally.

Reply:
@jamesricky you're right, we probably haven't thought this through...

> Should we maybe add an additional variant to the existing MUI Badge instead of creating our own?

I like this proposal. Is this possible?

> Maybe it makes more sense to just add a notificationCount: number prop to MenuItem, which does the rest internally.

I'm not so fond of having multiple props for specific features. Having one `secondaryAction` that's used for this seems more flexible to me. However, we could add a `ListItemBadge` component instead of having a custom `Badge` component.

How would you proceed here?

Reply:
> How would you proceed here?

- We should add docs for this component (and all other MUI component we prevent from being imported).
- This component doesn't allow overriding the MUI components `slotProps` -> this should be fixed.